Source: written by Pierre Lejeune <superheron@gmail.com>
Upstream: no
Reason: do src_install in ${IMAGE}/$(DESTDIR)
--- vsftpd-2.2.0/Makefile.orig	2009-05-22 21:44:52.000000000 +0200
+++ vsftpd-2.2.0/Makefile	2009-10-10 18:06:51.815302488 +0200
@@ -24,21 +24,21 @@
 	$(CC) -o vsftpd $(OBJS) $(LINK) $(LIBS) $(LDFLAGS)
 
 install:
-	if [ -x /usr/local/sbin ]; then \
-		$(INSTALL) -m 755 vsftpd /usr/local/sbin/vsftpd; \
+	if [ -x $(DESTDIR)/usr/local/sbin ]; then \
+		$(INSTALL) -m 755 vsftpd $(DESTDIR)/usr/local/sbin/vsftpd; \
 	else \
-		$(INSTALL) -m 755 vsftpd /usr/sbin/vsftpd; fi
-	if [ -x /usr/local/man ]; then \
-		$(INSTALL) -m 644 vsftpd.8 /usr/local/man/man8/vsftpd.8; \
-		$(INSTALL) -m 644 vsftpd.conf.5 /usr/local/man/man5/vsftpd.conf.5; \
-	elif [ -x /usr/share/man ]; then \
-		$(INSTALL) -m 644 vsftpd.8 /usr/share/man/man8/vsftpd.8; \
-		$(INSTALL) -m 644 vsftpd.conf.5 /usr/share/man/man5/vsftpd.conf.5; \
+		$(INSTALL) -m 755 vsftpd $(DESTDIR)/usr/sbin/vsftpd; fi
+	if [ -x $(DESTDIR)/usr/local/man ]; then \
+		$(INSTALL) -m 644 vsftpd.8 $(DESTDIR)/usr/local/man/man8/vsftpd.8; \
+		$(INSTALL) -m 644 vsftpd.conf.5 $(DESTDIR)/usr/local/man/man5/vsftpd.conf.5; \
+	elif [ -x $(DESTDIR)/usr/share/man ]; then \
+		$(INSTALL) -m 644 vsftpd.8 $(DESTDIR)/usr/share/man/man8/vsftpd.8; \
+		$(INSTALL) -m 644 vsftpd.conf.5 $(DESTDIR)/usr/share/man/man5/vsftpd.conf.5; \
 	else \
-		$(INSTALL) -m 644 vsftpd.8 /usr/man/man8/vsftpd.8; \
-		$(INSTALL) -m 644 vsftpd.conf.5 /usr/man/man5/vsftpd.conf.5; fi
-	if [ -x /etc/xinetd.d ]; then \
-		$(INSTALL) -m 644 xinetd.d/vsftpd /etc/xinetd.d/vsftpd; fi
+		$(INSTALL) -m 644 vsftpd.8 $(DESTDIR)/usr/man/man8/vsftpd.8; \
+		$(INSTALL) -m 644 vsftpd.conf.5 $(DESTDIR)/usr/man/man5/vsftpd.conf.5; fi
+	if [ -x $(DESTDIR)/etc/xinetd.d ]; then \
+		$(INSTALL) -m 644 xinetd.d/vsftpd $(DESTDIR)/etc/xinetd.d/vsftpd; fi
 
 clean:
 	rm -f *.o *.swp vsftpd
